migrations/1671102045930-create_audience_table.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class createAudienceTable1671102045930 implements MigrationInterface {
```

Comment:
```suggestion
export class CreateAudienceTable1671102045930 implements MigrationInterface {
```

Always make sure class name has Capital first letter

Reply:
done